### ISSUE TITLE
[RDP][Improvement] Add UI scaling percentage option (for Windows hosts)

### ISSUE THREAD

**Issue by andrigamerita** [iiordanov/remote-desktop-clients#492]
Hi, your RDP client is really the only one good on Android, but it's missing a quality-of-life and accessibility feature that's in my opinion much needed: custom UI scaling percentage setting.

Someone already made an issue about this but couldn't explain themselves and so it was closed (#396), that's why I'm creating this new issue.

Your client already allows for custom screen resolutions to be set, and that's usually fine (especially because that's the only thing that practically works on Linux hosts today) but sometimes you also need to be able to set the UI scale.

UI scale is different from resolution: altering the UI scale on Windows (on any OS where this concept is exposed really, but as I'm saying only Windows seems to support it via RDP) basically makes buttons, icons, window decorations, text etc... bigger, and that's fundamental to comfortably use a remote desktop on a device with small screen but high resolution (even the cheapest smartphones today have panels of an exaggerated resolution, for some reason). To give you an ideal example, my smartphone is 6.2" but more than 1920x1080 in resolution: connecting to Windows, some buttons appear smaller than the front camera hole!

Setting the screen resolution lower than native is a workaround, but it won't do as good things as being able to change the UI scale, since it makes everything on the screen blurry; UI scale is instead a setting that Windows takes care of internally.

Usually on Windows the UI scale can be set via Settings for phisical monitors, but unfortunately it's not possible for RDP sessions, as it expects the client to specify the desired UI scale percentage at connection; if the client doesn't allow the user to choose the UI scale (and doesn't guess it automatically from the client OS), then Windows will force it to 100% and, as I said, sometimes that makes everything on the screen way too small.

This feature could ideally be inplemented the same way the Microsoft Remote Desktop client for Android does, UI-side: expose a slider or text box to the user, decoupled from screen resolution, to set the scale.
Backend-side, see here in <https://learn.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/rdp-files> for how Microsoft implements the option in RDP files (and thus possibly in the actual RDP protocol? I don't know that as I'm not familiar with it): it uses the `desktopscalefactor:i:value` attribute.

Feel free to ask me for any other info that you would need about my issue, and thank you for your patience.
